src/Tasks/UpdateManifest.cs

```diff
@@ -35,4 +40,48 @@ public override bool Execute()
             return true;
         }
     }
+
+#else
+
+    public class UpdateManifest : TaskRequiresFramework, IUpdateManifestTaskContract
+    {
+        public UpdateManifest()
+            : base(nameof(UpdateManifest))
+        {
+        }
+
+        #region Properties
+
+        [Required]
```

Comment:
Question that I'm not sure about: should these attributes be on the interface? Should we drop them entirely from the stub version? It'd be annoying to be crafting an invocation of a target, working with a bunch of errors about how you must supply a value for whatever, only to eventually specify everything and then get "sorry that doesn't work on core"

Reply:
I will remove the `[Required]` attributes on the stub versions.

Attributes on an interface are not applied to a class implementing the interface.

The results of a quick test of moving the `[Output]` attribute from a class to an interface:

```
TaskALTest.proj(3,9): error MSB4803: The task "AL" is not supported on the .NET Core version of MSBuild. Please use the .NET Framework version of MSBuild. See https://aka.ms/msbuild/MSB4803 for further details.
TaskALTest.proj(4,11): error MSB4065: The "OutputAssembly" parameter is not marked for output by the "AL" task.
```

[TaskALTest.proj.txt](https://github.com/dotnet/msbuild/files/12466273/TaskALTest.proj.txt)

Note that the error for using a non-`Output` parameter as an output doesn't pre-empt the "not supported" error. In that respect, the `[Output]` attribute doesn't cause an issue.

Reply:
Removed `[Required]`